driver/src/main/java/org/neo4j/driver/internal/async/pool/NettyChannelTracker.java

```diff
@@ -95,6 +108,8 @@ public void channelClosed( Channel channel )
     {
         decrementIdle( channel );
         metricsListener.afterClosed( serverAddress( channel ) );
+
+        allChannels.remove( channel );
```

Comment:
Closed channels are automatically removed from a channel group. This line can be removed